sklearn/metrics/classification.py

```diff
@@ -950,7 +950,7 @@ def fbeta_score(y_true, y_pred, beta, labels=None, pos_label=1,
     y_pred : 1d array-like, or label indicator array / sparse matrix
         Estimated targets as returned by a classifier.
 
-    beta : float
+    beta : float, 0.0 <= beta <= 1.0
```

Comment:
why beta <= 1?

Reply:
@qinhanmin2014 Oops, sorry it's my mistake